bin/start_tests.sh

```diff
@@ -12,8 +12,8 @@ deactivate
 cd cli
 
 # overwrite core.dcos_url in the dcos.toml
-cp tests/data/dcos.toml $DCOS_CONFIG
-sed -i "s/change.dcos.url/$VBOX_IP/g" $DCOS_CONFIG
+cp tests/data/dcos.toml ${DCOS_CONFIG}
+sed -i "s/change.dcos.url/${CORE_DCOS_URL}/g" ${DCOS_CONFIG}
```

Comment:
Can we change this to `s/http:\/\/change.dcos.url/${CORE_DCOS_URL}/g` or something like that? I would like to be able to keep it as `dcos_url = http://...` in the `dcos.toml`.